.github/workflows/build.yml

```
@@ -25,7 +25,7 @@ jobs:
         if: matrix.platform.os == 'ubuntu-latest'
         run: |
           sudo apt-get update
-          sudo apt-get install -y webkit2gtk-4.0
+          sudo apt-get install -y webkit2gtk-4.1
```

I think we can publish a minor version with 2021 edition and using glib 0.16.
But we might need to stick in webkit2gtk-4.0 as many old distribution might not have 4.1
We'll look into 4.1 after we switch to flatpak for packaging and CICD.

```suggestion
          sudo apt-get install -y webkit2gtk-4.0
```